### DISCUSSION TITLE
How to convert PNG to sixel while keeping transparent part of image transparent?

### DISCUSSION THREAD

**Discussion by SherLock707** [hpjansson/chafa#234] [Q&A] [answered]
I use `foot` terminal and to display image as logo in `fastfetch` I needed to convert PNG image to `sixel` , I did it with following command: `chafa -s 32x17 image.png > image.sixel` 

It used to work fine, but with a foot update which "fixed" the wrong default background color for sixels, shows transparent part of PNG as black, How do I fix this?

Sixel: 
![image](https://github.com/user-attachments/assets/b8258c8b-d9a7-49e6-b63a-7b206b34cc90)

PNG:
![image](https://github.com/user-attachments/assets/5cf59107-a8e8-469c-8ef4-e4ca21c56a91)

**Comment by SherLock707** [comment]
For reference: https://codeberg.org/dnkl/foot/issues/1864

**Comment by hpjansson** [comment]
What does `chafa --version` say? Are you able to get any transparency if you invoke it with `-t 0.5`?

**Reply by SherLock707** [reply]
```
Chafa version 1.14.2

Loaders:  AVIF GIF JPEG JXL PNG QOI SVG TIFF WebP XWD
Features: mmx sse4.1 popcnt avx2
Applying: mmx sse4.1 popcnt avx2
```

`foot version: 1.19.0 +pgo +ime +graphemes +toplevel-icon -assertions`

`fastfetch 2.30.1 (x86_64)`

**Reply by hpjansson** [reply, marked as answer]
You need at least Chafa version 1.14.3 -- maybe you can poke your distro maintainer to update it. The latest is version 1.14.5. See the changelog at https://github.com/hpjansson/chafa/blob/master/NEWS .

**Reply by SherLock707** [reply]
Oh understood, I'll see what I can do about it. thanks.

**Reply by hpjansson** [reply]
Np! We all had to adjust how we were doing sixels a bit over the last year, but the good thing is that after the discussions we have a much stronger consensus than before - so less breakage going forward.

**Reply by SherLock707** [reply]
Updated from AUR and its working, thanks!

![image](https://github.com/user-attachments/assets/505a1bad-17ce-4d5e-810c-4354f84b6e88)
